src/wallet/rpcwallet.cpp

```diff
@@ -5014,17 +5014,20 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
             "z_sendmany \"fromaddress\" [{\"address\":... ,\"amount\":...},...] ( minconf ) ( fee ) ( privacyPolicy )\n"
             "\nSend multiple times. Amounts are decimal numbers with at most 8 digits of precision."
             "\nChange generated from one or more transparent addresses flows to a new transparent"
-            "\naddress, while change generated from a shielded address returns to itself."
-            "\nWhen sending coinbase UTXOs to a shielded address, change is not allowed."
-            "\nThe entire value of the UTXO(s) must be consumed."
-            + strprintf("\nBefore Sapling activates, the maximum number of zaddr outputs is %d due to transaction size limits.\n", Z_SENDMANY_MAX_ZADDR_OUTPUTS_BEFORE_SAPLING)
+            "\naddress, while change generated from a legacy Sapling address returns to itself."
+            "\nWhen sending from a unified address, change is returned to the internal-only address"
+            "\nfor the associated unified account."
+            "\nWhen spending coinbase UTXOs, only shielded recipients are permitted and change is not allowed;"
+            "\nthe entire value of the coinbase UTXO(s) must be consumed."
```

Comment:
Why the leading "\n" in this section? (Or reverse the question for the other section.) I'm fine with addressing the inconsistency in a later commit and I know you didn't introduce it here, just curious if there's some motivation.

Reply:
No motivation I'm aware of.

Reply:
IIRC leading `\n` is most common, particularly in RPC methods inherited from upstream. We need at least one because this all relies on string constant concatenation.